modules/openthread/platform/radio_nrf5.h

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright (c) 2025 Nordic Semiconductor ASA
```

Comment:
I wonder whether we should have this function available from Zephyr, somewhere here: https://github.com/nrfconnect/sdk-zephyr/tree/main/modules/openthread. I'm thinking about the implementation with the L2 layer as well for this function.

Reply:
But actually, I don't have a strong opinion about it. If you think that it will be enough to have it only for raw NRF implementation, it's OK as it is :)

Reply:
I like the idea to have a well-defined public API for this. Let me try to add this.

Reply:
@ArekBalysNordic Would it be ok to merge this PR in the current form and deliver changes as a followup? As contribution to Zephyr will take some time and we have other work depending on this one :)

Reply:
Sure :)